### ISSUE TITLE
Hand Menu example eye tracking does not function well over Holographic Remoting

### ISSUE THREAD

**Issue by elbuhofantasma** [microsoft/MixedRealityToolkit-Unity#9593]
## Describe the bug

When the user is in the Hand Menu example scene over Holographic Remoting and they raise their flat palm into view of the device sensors without looking at their palm, the hand menu will populate on the user's flat palm. This issue only happens with XR SDK and Holographic Remoting.  I do not know if this issue effects OpenXR as well since articulated hands do not work over Holographic Remoting.

## To reproduce

1. Create a new project, import the MRTK, and enable the XR SDK
2. Open the Hand Menu Examples scene
3. Change to the 'DefaultHoloLens2XRSDKConfigurationProfile' 
4. Start the Holographic Remoting session
5. Raise a flat palm into view of the device sensors but do not look in the direction of your flat palm
6. See the hand menu populate on your flat palm, even though your gaze was not near your flat palm

## Your setup (please complete the following information)

- Unity Version 2020.3.2f1
- MRTK Version 2.5.1

## Target platform (please complete the following information)

- HoloLens 2
- UWP
- XR SDK

**Comment by polar-kev** [comment]
By default eye tracking is turned off in the XR SDK profile while we wait on some Unity bugs to be fixed.

**Comment by IssueSyncBot** [comment]
We appreciate your feedback and thank you for reporting this issue.

Microsoft Mixed Reality Toolkit version 2 (MRTK2) is currently in limited support. This means that Microsoft is only fixing high priority security issues. Unfortunately, this issue does not meet the necessary priority and will be closed. If you strongly feel that this issue deserves more attention, please open a new issue and explain why it is important.

Microsoft recommends that all new HoloLens 2 Unity applications use MRTK3 instead of MRTK2.

Please note that MRTK3 was released in August 2023. It features an all new architecture for developing rich mixed reality experiences and has a minimum requirement of Unity 2021.3 LTS. For more information about MRTK3, please visit[https://www.mixedrealitytoolkit.org](https://www.mixedrealitytoolkit.org).

Thank you for your continued support of the Mixed Reality Toolkit!
